### PR TITLE
Minecraft: Setting Up Crossplay

### PR DESCRIPTION
**Pull request by TriggeredLegend**
- I've written a detailed guide on how to set up crossplay on your Minecraft server from ZAP-Hosting, in line with my approved suggestion #1369

### CODE REVIEW

**Comment by ThatGuyJacobee** [comment]
Hi there, thank you for submitting your contribution.

In general, the guide did achieve the goal of exploring the process of setting up GeyserMC for crossplay. However, there was some lack of detail in various sectors which could have been expanded. Likewise some of the things didn't exactly fit into the guidelines that we utilise. I've done the necessary adjustments to improve upon this.

Your PR has been accepted. Please let me know whether you would prefer a ZAP Coins or PayPal payout. My colleague will perform some extra checks before merging your content into the docs once the payment is finalised.

Kind Regards.

**Review by ThatGuyJacobee — APPROVED** [review]

**Comment by TriggeredLegend** [comment]
Hey @ThatGuyJacobee !!!
- I prefer to receive it via PayPal. Thank you! I will wait for the final confirmation.

**Comment by ThatGuyJacobee** [comment]
Hey there @TriggeredLegend! We accept your final submission for this contribution and thank you for your interest and participation in the Contribution Program.

We have sent a PayPal payout confirmation ticket to your provided ZAP-Hosting Account. You must now send us an invoice using the PayPal Invoice Creator, including the ticket # number of the payout confirmation in the description. Please read our [guide](https://zap-hosting.com/guides/docs/contribution-rewards/#paypal) for more information.

Once you have sent your invoice, it may take up to a few days to be processed. Thank you for contributing and we hope to see you contribute more in the future! 💚

**Pull request merged**
